boards/nrf52833_dk.json

```diff
@@ -0,0 +1,51 @@
+{
+  "build": {
+    "arduino": {
+      "ldscript": "nrf52_xxaa.ld"
+    },
+    "core": "nRF5",
+    "cpu": "cortex-m4",
+    "extra_flags": "-DARDUINO_NRF52_DK",
+    "f_cpu": "64000000L",
+    "mcu": "nrf52833",
+    "variant": "nRF52DK",
+    "zephyr": {
+      "variant": "nrf52833dk_nrf52833"
```

Comment:
I don't see this board in the list of supported hardware here https://github.com/sandeepmistry/arduino-nRF5
Even though you can use the `nRF52DK` variant as a substitute, it's still a different board, possibly with different pinout which will confuse users when their "blink" sketch doesn't work. I propose to delete the `arduino`, `core`, `extra_flags` and `variant` fields.